docs/view_components/welcome.rst

```diff
@@ -0,0 +1,38 @@
+linking ************
+Welcome page
+************
+
+A welcome window can be seen in almost every software/website in production. The welcome window should serve as an orientation point for the user and let them know, in as brief words (and other audiovisual content) as possible, what to expect from the tool. It is also the launch point from where the user begins their energy systems simulation.
+
+The welcome page is to be implemented as an optional pop-up that springs up on the landing page when the user launches the tool. In this section, a structure for the welcome page is proposed, subject to further discussion with the stakeholders.
+
+Attributes
+----------
+
+The following are the view-components available to the user in the welcome pop-up on the landing window immediately after launching the software for the first time:
```

Comment:
The view-component is the welcome view. I agree that a label or a link to a video are also "view-components" in some sense, but there are very generic or elementary. We want to describe here the "view-components" of our GUI which is a bit more specific.

Reply:
I propose that we separate the term view-component into View and Components. Right now it is confusing. 

View = components + the way they are rendered (here, Welcome view)
Components = all the elements in the view such as pop-ups, links
Rendering = how these things are rendered (the welcome view is rendered as a pop-up on the landing page)